### ISSUE TITLE
Windows always stuck

### ISSUE THREAD

**Issue by jsh088** [meganz/MEGAsync#302]
First it gets stuck on "logging in"
If I keep messing with it, it will eventually go through. But then it gets stuck on uploads and keeps failing.
Has worked without issues until this year, possibly when I switched over to Windows 10.

**Comment by d4rkd0s** [comment]
I have the exact same issue. My client is "not responding" I'm going to try to reinstall.

**Comment by ZanPotokar** [comment]
Did reinstall solve your issue?

**Comment by d4rkd0s** [comment]
Yes! It actually did @ZanPotokar so I did a standard "Uninstall" on the "Add and remove programs" on Windows. I also rebooted after uninstall, and rebooted after install. Just to be safe. But it's all working good now. I think it's just some cache that builds up, since I'm moving a lot of data, about 1.3TB or 80,000 files so I'm expecting to maybe do this again. Not sure, been smooth so far, 2 days now.

**Comment by d4rkd0s** [comment]
I believe @vtmateos and @polmr are the current maintainers. If they want, I can gather more information on windows, debug this if I can. Qt is pretty heavy but I'll try to setup local dev and get some more info for this Windows "slowness"

**Comment by jsh088** [comment]
reinstall did not solve my problem, but it has been working at least some times these days.

**Comment by vtmateos** [comment]
Hi there,
Apologies for the issues you are getting while using MEGAsync. They could be related with temporary issues with servers, issues with local synced files, permissions, etc. We'd need you contact our support team at support@mega.nz so we can investigate the culprit of the problems and request from you the needed information.

Thanks.

**Comment by d4rkd0s** [comment]
Yeah I'm also having issues from time to time. I think the application has too many "blocks" on file system access. It's typically during indexing / adjustments to my filesystem, the Mega app locks up as it catches up, this should be a backgrounded process so the UI works and responds to the user. This is only my wild guess. I'll reach out to support@mega.nz in the future thank you @vtmateos

**Comment by jsh088** [comment]
yea its not working at all now.
Was stuck on Loggin in for a while, then I restarted the computer and its now stuck at "Waiting".
Exact same problem as before.

**Comment by rl-mega** [comment]
Please re-open the ticket if the issue persists with the latest release.
